news/2020-04-07-mappers-report-march.md

```diff
@@ -0,0 +1,156 @@
+
+So far in the contest, [Jemzuu](https://osu.ppy.sh/users/7890134) has found himself at the top of the podium during the first 2 rounds, quite impressive! Good luck to those participating in the approaching rounds and we hope to see more novel well organized contests like this in the future.
+
+### Monthly Beatmapping Contest: February 2020 Results
+
+We’d like to congratulate [Aphestra](https://osu.ppy.sh/users/11949191) on their win in the 2nd Monthly Beatmapping Contest, earning them a profile badge! Good job as well to [UndeadCapulet](https://osu.ppy.sh/users/2523533) and [Mao](https://osu.ppy.sh/users/2204515) who were the community vote runner-up and screening crew favourite respectively. Both of them find themselves halfway to a beautiful profile badge thanks to their placements. Full results for the contest can be found [here!](https://osu.ppy.sh/home/news/2020-03-21-monthly-beatmapping-contest-2-results) 
+
+### Monthly Beatmapping Contest: April 2020
+
+Another month, another monthly beatmapping contest! Head over to the [news post](https://osu.ppy.sh/home/news/2020-04-04-monthly-beatmapping-contest-april) to find out all about this month’s restrictions! Additionally the community voting phase of the March beatmap contest has started so be sure to go vote on the [contest listing](https://osu.ppy.sh/community/contests/93)!
+
+## <a id=”around-community”></a>Around the Community
+
+Another new section this month! All around the community. Whenever something interesting happens in the community which does not fit into any category above we will include that here. Should you have/know about something interesting that’s happening within the mapping community, do not hesitate to let [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323) know about it! 
+
+### The osu! Map Cast
+
+Starting up 2 months ago, there’s been a new podcast series hosted by [Cinnabun](https://osu.ppy.sh/users/11868659) titled the **osu! Map Cast**. This podcast is a series hosted on YouTube where Cinnabun talks with mappers, BNs, and NATs about their views on mapping, modding, and other related topics. With 8 episodes out and new episodes released weekly it will only further grow from here. In the episodes released so far, they’ve talked to the guests about osu! as well as their own personal experiences, so be sure to check it out if you want to learn something new or just want to listen to something osu! related. You can find out more about the osu! Map Cast [here](https://osu.ppy.sh/community/forums/topics/1043221)!
```

Comment:
```suggestion
Starting up 2 months ago, there’s been a new podcast series hosted by [Cinnabun](https://osu.ppy.sh/users/11868659) titled the **osu! Map Cast**. This podcast is a series hosted on YouTube where Cinnabun talks with mappers, BNs, and NATs about their views on mapping, modding, and other related topics. With 9 episodes out and new episodes released weekly it will only further grow from here. In the episodes released so far, they’ve talked to the guests about osu! as well as their own personal experiences, so be sure to check it out if you want to learn something new or just want to listen to something osu! related. You can find out more about the osu! Map Cast [here](https://osu.ppy.sh/community/forums/topics/1043221)!
```